app/javascript/react/pages/CalendarPage.tsx

```diff
@@ -4,11 +4,13 @@ import styled from "styled-components";
 
 import { Navbar } from "../components/Navbar/Navbar";
 import { Graph } from "../components/Graph";
-import { grey100, white } from "../assets/styles/colors";
+import { white } from "../assets/styles/colors";
 import { CalendarStationHeader } from "../components/molecules/CalendarStationHeader/CalendarStationHeader";
+import { WeekView } from "../components/WeekView/WeekView";
+import { thresholdsValues, weeklyData } from "../components/WeekView/WeeklyMockData";
 
 const PageLayout = styled.div`
-  background-color: ${grey100};
+  background-color: ${white};
   height: 100vh;
   padding: 30px;
 `;
```

Comment:
is it just for testing or foreva? 🫠 If it stays can you make `CalendarPage.style.tsx` for that?